### ISSUE TITLE
Headerbar color demo doesn't work

### ISSUE THREAD

**Issue by danrabbit** [elementary/granite#556]
### What Happened?

Since porting to GTK4, the demo for changing headerbar color doesn't seem to have any effect

### Steps to Reproduce

Set a custom headerbar color on the "Style Classes" page and notice it doesn't change

### Expected Behavior

Headerbar color should change

### OS Version

7.x (Early Access)

### Software Version

Compiled from git

**Comment by davidmhewitt** [comment]
Yes, this is one of my known regressions from #542 

I noted:
> Need to fix Granite.Widgets.Utils.set_color_primary 

I think it was using deprecated APIs internally, and will possibly depend on how we want to do this in the stylesheet? So I just commented it out.

**Comment by danrabbit** [comment]
Yeah I just wanted to make sure I was tracking these things somewhere I won't forget to follow up on it :)
